README.md

```diff
@@ -1,10 +1,12 @@
 ## OpenShift 4.2 Designs
 Access design documentation for features slotted in the 4.2 release of OpenShift. Click links below for specific section details.
 
+- [API Explorer](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/api-explorer/api-explorer)
 - [Command Line Tools](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/command-line-tools/command-line-tools)
 - [Console Customizations](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/console-customizations/console-customizations)
 - [Contextual Help](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/contextual-help/contextual-help)
 - [Dashboards and Projects](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/dashboards/dashboards)
+- [Diagnostics (WIP)](https://docs.google.com/document/d/11rQliTE7fdyqDWFc5F-w-jCBRxxNBymngidQ6xlCiws/edit?usp=sharing)
```

Comment:
@bmignano - If you want to break this out into a separate PR I think we can merge it right away. Up to you though if you are stuck in the API Explorer branch for some reason :)

Reply:
@lizsurette That Diagnostics one is still a WIP so we opted to remove the detailed documentation for now and @bmignano is just linking to a GDoc while we keep the WIP title. Do you think we need to pull that?

Reply:
> @lizsurette That Diagnostics one is still a WIP so we opted to remove the detailed documentation for now and @bmignano is just linking to a GDoc while we keep the WIP title. Do you think we need to pull that?

Nope totally cool with the addition! Just didn't know if the Explorer designs PR that it's now lumped in would get held up longer or not. Looks like not since you approved :) I'll approve too :)